src/clr-angular/forms/styles/_variables.forms.scss

```diff
@@ -79,3 +79,12 @@ $clr-forms-radio-top: $clr-forms-checkbox-top;
 $clr-forms-radio-selected-shadow: $clr-forms-checkbox-checked-shadow;
 $clr-forms-radio-focused-shadow: 0 0 $clr-outline-blur $clr-outline-spread $clr-outline-color;
 $clr-forms-radio-disabled-shadow: $clr-forms-checkbox-checked-shadow;
+
+// Range
+$clr-forms-range-progress-fill-color: $clr-color-action-600 !default;
+$clr-forms-range-track-color: $clr-color-neutral-200 !default;
+$clr-forms-range-track-height: 4px; //$clr_baselineRem_4px; //4px
```

Comment:
The `rem` units are not rendered correctly in Edge and IE11.
![picturemessage_025gjks1 a5c](https://user-images.githubusercontent.com/53469024/66834039-99a56700-ef65-11e9-933e-6e4c499679d0.png)
In this picture, the width of the circle is set in pixels and the height is set in rem units. In any other browsers, they are equal.

Reply:
`rem` are weird with sliders. But we can't use hard pixel values. It would mean that we are committing a change that breaks a huge amount of work we've done to enable accessible resizing of the browser content through browser settings.

Basically, everything _except this one thing_ would resize.

I recommend looking into percent-based values.